packages/base/contains-many-component.gts

```diff
@@ -46,14 +54,51 @@ interface ContainsManyEditorSignature {
 }
 
 class ContainsManyEditor extends GlimmerComponent<ContainsManyEditorSignature> {
+  private sortableGroupId: string;
```

Comment:
Using a UUID for groupName prevents the sorting mechanism from breaking during re-rendering.